.gitignore

```diff
@@ -82,3 +82,6 @@ bionic/
 focal/
 jammy/
 release/
+.vscode/launch.json
+.vscode/settings.json
+.vscode/tasks.json
```

Comment:
This deserves its own commit.  Maybe at the start?  Actually, I think you can ignore .vscode/*?